pass-bakery/app/controllers/HomeController.scala

```diff
@@ -21,4 +21,5 @@ class HomeController @Inject()(val controllerComponents: ControllerComponents) e
   def index() = Action { implicit request: Request[AnyContent] =>
     Ok(views.html.index())
   }
+
```

Comment:
Not needed for your current changes. Keep your diff organized to make reviewers have an easier time reading your PRs